src/gl/gl3device.cpp

```diff
@@ -1763,7 +1841,58 @@ startSDL3(void)
 		return 0;
 	}
 
+#if !defined(__ANDROID__) && !defined(ANDROID)
 	printf("OpenGL version: %s\n", glGetString(GL_VERSION));
+#else
+	char logPath[256];
+	sprintf(logPath, "%s/GL_info.txt", getenv("GAMEFILES"));
+	FILE* log = fopen(logPath, "w");
```

Comment:
Wouldn't it be more useful and less prone to bugs (when e.g. `GAMEFILES` is not defined) by using `SDL_Log`?
You can then view this data using logcat.
(Same applies to `startSDL2`)

Also perhaps this can be shared between SDL2 and SDL3? Using `#if defined(LIBRW_SDL2) || defined(LIBRW_SDL3)